src/lib.rs

```diff
@@ -302,6 +302,15 @@ impl fmt::Display for LicenseItem {
     }
 }
 
+#[derive(Copy, Clone, PartialEq)]
+#[non_exhaustive]
+pub enum ParseMode {
+    /// Strict SPDX parsing
+    Default,
```

Comment:
Maybe this should be called Strict or Exact instead of Default, the user won't necessarily know what Default means.